### ISSUE TITLE
Package that failed validation is visible under "Published Packages" heading

### ISSUE THREAD

**Issue by joelverhagen** [NuGet/NuGetGallery#5375]
If all versions of a package are in the validating or failed validation state, they should not be under a heading called "Published Packages"

## Manage packages page

![image](https://user-images.githubusercontent.com/94054/35599582-3705852a-05de-11e8-95fc-d073a3b249d6.png)
![image](https://user-images.githubusercontent.com/94054/35599590-427750f0-05de-11e8-8f6f-38bfd7cbba62.png)

# Package details page

![image](https://user-images.githubusercontent.com/94054/35599596-486834a2-05de-11e8-9e7e-104557774222.png)

**Comment by joelverhagen** [comment]
The page is not published 😄

**Comment by xavierdecoster** [comment]
This view currently only queries for the "latest" package version and splits on the `Listed` property.

Is it even possible for a package version to be marked as "latest", "listed" and have a status of either "validating", "validation failed", or "deleted", all at the same time?
Should a package registration for which all versions failed validation even have a package marked as "listed" or "latest"?

In other words: is it safe to assume that if the "latest" package doesn't have a status of "Available", all other versions also are unavailable?

**Comment by xavierdecoster** [comment]
@joelverhagen can you please verify whether the Zip64Package is shown under the unlisted packages section on your account page?
